sunpy/coordinates/great_arc.py

```diff
@@ -0,0 +1,121 @@
+#
+# Calculates the co-ordinates along great arcs between two specified points
+# which are assumed to be on disk.
+#
+from __future__ import absolute_import, division, print_function
+
+import numpy as np
+from astropy.coordinates import SkyCoord
+import sunpy.coordinates
+
+
+def great_arc(start_point, end_point, center=None, number_points=100):
+    """
+    Calculate a user-specified number of points on a great arc between a start
+    and end point on a sphere.
+
+    Parameters
+    ----------
+    start_point : `~astropy.coordinates.SkyCoord`
+        Start point.
+
+    end_point : `~astropy.coordinates.SkyCoord`
+        End point.
+
+    center : `~astropy.coordinates.SkyCoord`
+        Center of the sphere.
+
+    number_points : int
+        Number of points along the great arc.
+
+    Returns
+    -------
+    arc : `~astropy.coordinates.SkyCoord`
+        Co-ordinates along the great arc in the co-ordinate frame of the
+        start point.
+
+    Example
+    -------
+    >>> from astropy.coordinates import SkyCoord
+    >>> import astropy.units as u
+    >>> import sunpy.coordinates
```

Comment:
Yes it is imported along with sunpy.map. I am torn if we should import it here. actually importing great_arc will register the frames. We don't need it.